esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -1111,6 +1231,190 @@ def _plot_zonal_mean_profile_without_ref(self, plot_func, dataset):
 
         return (plot_path, {netcdf_path: cube})
 
+    def _plot_hovmoeller_z_vs_time_without_ref(self, plot_func, dataset):
+        """Plot hovmoeller z vs time for single dataset without reference."""
+        plot_type = 'hovmoeller_z_vs_time'
+        logger.info(
+            "Plotting homvoeller z vs time without reference dataset"
+            " for '%s'", self._get_label(dataset))
+
+        # Make sure that the data has the correct dimensions
+        cube = dataset['cube']
+        dim_coords_dat = self._check_cube_dimensions(cube, plot_type)
+
+        time_coord = cube.coord(axis='T')
+
+        # Create plot with desired settings
+        with mpl.rc_context(self._get_custom_mpl_rc_params(plot_type)):
+            fig = plt.figure(**self.cfg['figure_kwargs'])
+            axes = fig.add_subplot()
+            plot_kwargs = self._get_plot_kwargs(plot_type, dataset)
+            plot_kwargs['axes'] = axes
+            plot_zonal_mean_profile = plot_func(cube, **plot_kwargs)
+
+            # Print statistics if desired
+            self._add_stats(plot_type, axes, dim_coords_dat, dataset)
+
+            # Setup colorbar
+            fontsize = self.plots[plot_type]['fontsize']
+            colorbar = fig.colorbar(plot_zonal_mean_profile,
+                                    ax=axes,
+                                    **self._get_cbar_kwargs(plot_type))
+            colorbar.set_label(self._get_cbar_label(plot_type, dataset),
+                               fontsize=fontsize)
+            colorbar.ax.tick_params(labelsize=fontsize)
+
+            # Customize plot
+            axes.set_title(self._get_label(dataset))
+            fig.suptitle(f"{dataset['long_name']} ({dataset['start_year']}-"
+                         f"{dataset['end_year']})")
+            z_coord = cube.coord(axis='Z')
+            axes.set_ylabel(f'{z_coord.long_name} [{z_coord.units}]')
+            if self.plots[plot_type]['log_y']:
+                axes.set_yscale('log')
+                axes.get_yaxis().set_major_formatter(
+                    FormatStrFormatter('%.1f'))
+            if self.plots[plot_type]['show_y_minor_ticklabels']:
+                axes.get_yaxis().set_minor_formatter(
+                    FormatStrFormatter('%.1f'))
+            else:
+                axes.get_yaxis().set_minor_formatter(NullFormatter())
+            axes.get_xaxis().set_major_formatter(
+                mdates.DateFormatter(self.plots[plot_type]['time_format']))
```

Comment:
```suggestion
            if self.plots[plot_type]['time_format'] is not None:
                axes.get_xaxis().set_major_formatter(
                    mdates.DateFormatter(self.plots[plot_type]['time_format']))
```

To make sure that the default formatter can be used if desired.